test/integ/test_put_get.py

```diff
@@ -621,3 +621,20 @@ def fake_cmd_query(*a, **kw):
     downloaded_file = get_dir / upload_file.name
     assert downloaded_file.exists()
     assert filecmp.cmp(upload_file, downloaded_file)
+
+
+@pytest.mark.skipolddriver
+def test_put_special_file_name(tmp_path, conn_cnx):
+    test_file = tmp_path / "data~%23.csv"
+    test_file.write_text("1,2,3\n")
+    stage_name = random_string(5, "test_special_filename_")
+    with conn_cnx() as con:
+        with con.cursor() as cur:
+            cur.execute(f"create temporary stage {stage_name}")
+            cur.execute(
+                "PUT 'file://{}' @{}".format(
+                    str(test_file).replace("\\", "/"), stage_name
+                )
```

Comment:
```suggestion
                f"PUT file://{str(test_file).replace('\\', '/')} @{stage_name}"
```

Reply:
Furthermore; I'm not sure if the replace is necessary, I think our back-end is able to handle both Windows and Unix paths, could you check?

Reply:
From test_put_windows_path, replace() seems not necessary.